docs/user/exercises/modeling.rst

```diff
@@ -1,2 +1,224 @@
+
+    .. figure:: modeling/create-modeling-exercise-form-3.png
+              :align: center
+
+
+Once you are done defining the schema of an exercise, you can now create an exercise by clicking on |save| button.
+You will then be redirected to **Example Submissions for Assessment Training** Page.
+
+    .. figure:: modeling/example-submission-for-assessment-training.png
+              :align: center
+
+In this page, you can either *Create Example Submission* or *Use as Example Submission* for Assessment Training.
+Example submissions can be used to assess the submissions of students semi-automatically.
+Artemis uses those submissions to automatically apply the known assessment comments to similar model elements in other submissions as well.
+
+- Select |create-example-submission| if you want to create an example submission from scratch.
+- Alternatively, after the exercise already started, you can also use some submissions submitted by students as an example submission. For that, click on |use-as-example-submission|.
+
+
+.. note::
+    Artemis uses semi-automatic grading of modeling exercises using machine learning.
+    You can hence train the model by selecting *Use in Assessment Training* checkbox while creating an example submission.
+
+    .. figure:: modeling/use-in-assessment-training.png
+              :align: center
+
+Import Modeling Exercise
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Alternatively, you can also import modeling exercise from the existing one by clicking on **Import Modeling Exercise**.
+
+    .. figure:: modeling/import-modeling-exercise.png
+              :align: center
+
+- An import modal will prompt up, where you will have an option to select and import previous modeling exercises from the list by clicking on |import| button.
+
+    .. figure:: modeling/import-modeling-exercise-modal.png
+              :align: center
+
+- Once you import one of the exercise, you will then be redirected to a form which is similar to *Create new modeling exercise* form with all the fields filled from imported exercise. You can now modify the fields as per your necessity to create a new Modeling Exercise.
+
+Result
+^^^^^^
+
+    .. figure:: modeling/course-dashboard-exercise-modeling.png
+              :align: center
+
+- Click the |edit| button of the modeling exercise and adapt the interactive problem statement. There you can also set release and due dates.
+- Click the |scores| button to see the scores achieved by the students.
+- Click the |participation| button to see the list of students participated in the exercise.
+- Click the |submission| button to see the list of submission submitted by students.
+- Click the |example-submission| button to modify/add example submission of the exercise.
+- Click the |delete| button to delete the exercise.
+- You can get an overview of the exercise by clicking on the title.
+
+Student Submission
```

Comment:
Maybe move this to an own subpage (and the rest into an other?)
This way, a students don't have to read through half the page before finding the relevant information for them